client/components/editor/ActivityContent/index.vue

```diff
@@ -148,8 +148,9 @@ export default {
 
 <style lang="scss" scoped>
 .activity-content {
+  flex-grow: 1;
   min-height: 100%;
-  padding: 4.375rem 1.5625rem 0 26.25rem;
```

Comment:
We lost spacing on lower resolution =>
![image](https://user-images.githubusercontent.com/6253820/101055594-54d5f880-358a-11eb-8dd0-cd1a8d1f7aa3.png)